Taskfile.yml

```
@@ -73,6 +73,26 @@ tasks:
     cmds:
       - docker compose down --volumes --remove-orphans
 
+  brew:
+    preconditions:
+      - sh: brew -v
+        msg: 'This task requires `brew`. Please refer to this documentation: https://brew.sh/'
+
+  pre-commit:
+    desc: Install pre-commit tool
+    internal: true
+    deps: [brew]
+    status:
+      - pre-commit --version
+    cmds:
+      - brew install pre-commit
+
+  install:pre-commit:
```

Use this to have proper namespacing
```suggestion
  pre-commit:install
```

---

Thanks, done